rclcpp/include/rclcpp/node_interfaces/node_graph_interface.hpp

```diff
@@ -25,10 +26,40 @@
 
 #include "rclcpp/event.hpp"
 #include "rclcpp/macros.hpp"
+#include "rclcpp/qos.hpp"
 #include "rclcpp/visibility_control.hpp"
+#include "rmw/topic_endpoint_info.h"
 
 namespace rclcpp
 {
+
+using EndpointType = rmw_endpoint_type_t;
```

Comment:
It would be good to define an `enum class`, instead of just using de underlying `enum`.

Reply:
Done.